### PR TITLE
hero must stay still to show indicator + indicator made more visible

### PR DESCRIPTION
**Pull request by duralakun**
This enables you to set a time the hero must stay still until he can see the flag indicator.
If you set the time to 0 the flag indicator will always be shown.
The bullet was hard to see and you couldnt see it behind walls. This is why i changed it to a laser dot and also added a laser line that will appear for half a second indicating to you where the flag is.
#95

### CODE REVIEW

File: src/game/server/entities/character.cpp [hunk 1]
```diff
@@ -3160,22 +3160,39 @@ void CCharacter::Snap(int SnappingClient)
...
 				pObj->m_X = (int)Indicator.x;
 				pObj->m_Y = (int)Indicator.y;
-				pObj->m_VelX = 0;
-				pObj->m_VelY = 0;
+				pObj->m_FromX = pObj->m_X;
+				pObj->m_FromY = pObj->m_Y;
 				pObj->m_StartTick = Server()->Tick();
+
+				// display laser beam for 0.5 seconds
+				if (g_Config.m_InfHeroFlagIndicatorTime > 0 && Server()->Tick()-inactiveTicks < Server()->TickSpeed()*0.5)
```
Comment (bretonium) [inline src/game/server/entities/character.cpp]: > `if (g_Config.m_InfHeroFlagIndicatorTime > 0 ...)`

Maybe 0 could mean that indicator should always be displayed?

Comment (duralakun) [reply src/game/server/entities/character.cpp]: Yes that is the case, but the laser line will not be shown. Only the laser dot.

**Comment by duralakun** [comment]
I again improved the visual effect, it looks way better now :D
I agree it would be nice if the hero only has to stand, but can still move the mouse, need to create a new counter for this.

**Comment by duralakun** [comment]
bretonium could you maybe make the help text when you write /help hero, if you have time
I have never done this and you seem to have more experience with it.

**Comment by duralakun** [comment]
The indicator will now be shown even if the hero shoots or looks around, only requirement is to not move. 
I just realized that this only depends on the hero input. So if someone else moves the hero (with grenades or zombie hooks him) it will not change the counter. But i dont think that this is a problem.

**Pull request merged**